src/php/inc/theme-setup.php

```diff
@@ -68,7 +68,8 @@ function sparkpress_theme_setup() {
 function custom_hide_editor() {
 	global $pagenow;
 	$hidden_pages = array(
-		'example-page.php',
+		// specify page templates where main editor should be hidden
+		// 'example-page-template.php',
```

Comment:
I don't believe that `example-page.php` exists, but I also didn't want to hide the editor for something by default only to have someone wonder why they can't edit a page with that template, so this felt like a happy medium to me.